skipruntime-ts/server/src/server.ts

```diff
@@ -5,11 +5,43 @@ import {
 } from "skip-wasm";
 import { createRESTServer } from "./rest.js";
 
+/**
+ * Initialize and start a service.
+ *
+ * Calling `runService` will initialize a service based on the `service` specification and start listening for requests on the specified `port`.
+ * This is a REST service that provides access to the internal reactive service.
+ * The service exposes some resources specified by a {@link SkipService}.
+ * Each resource has a _name_ and identifies a collection that associates _keys_ to _values_.
+ * The service responds to the following HTTP requests:
+ * - Read: `GET /v1/:resource/:key?param1=value1&...&paramN=valueN`
+ *
+ *   Instantiate the named `resource` with parameters `{param1=value1,...,paramN=valueN}` and respond with the values associated with the given `key`. If the named resource has previously been instantiated with the same parameters, then it is reused.
+ *
+ * - Subscribe: `GET /v1/:resource?param1=value1&...&paramN=valueN`
```

Comment:
Currently requires an `Accept: text/event-stream` header, otherwise you're not getting an SSE stream but a simple JSON document representing a snapshot of the collection at current time.